src/input/source/evdev.rs

```diff
@@ -133,6 +156,9 @@ impl EventDevice {
         if properties.contains_key("ID_INPUT_JOYSTICK") {
             return DriverType::Gamepad;
         }
+        if properties.contains_key("ID_INPUT_KEYBOARD") {
```

Comment:
This has the potential to reek havok on devices already using keyboard input for extra buttons. What is the behavior if no config is specified for a legacy evdev atkbd?